packages/leancode_contracts_generator/lib/src/generator_script.dart

```diff
@@ -5,14 +5,22 @@ import 'contracts_generator_exception.dart';
 /// Interface to the C# protobuf generator
 class GeneratorScript {
   /// Generate for a list of projects. Passed paths need to point to a .csproj file.
-  GeneratorScript.project(List<String> projects)
-      : args = ['project', '-p', ...projects];
+  GeneratorScript.project(
+    List<String> projects, {
+    List<String>? options,
+  }) : args = [
+          'project',
+          '-p',
+          ...projects,
+          if (options != null) ...options.map((o) => '--$o')
```

Comment:
Instead of prepending `--` here and below, just have the user specify the flag with preceding `--` in the config file, otherwise this will prevent users from passing short form flags (e.g. `-k`).